stdlib/public/SwiftShims/CoreFoundationShims.h

```diff
@@ -69,6 +69,7 @@ _swift_shims_CFIndex _swift_stdlib_CFStringGetLength(
     _swift_shims_CFStringRef _Nonnull theString);
 
 SWIFT_RUNTIME_STDLIB_INTERFACE
+__attribute__((ns_returns_retained))
```

Comment:
"release" of course. Here https://github.com/apple/swift/blob/swift-4.0-branch/stdlib/public/core/StringBridge.swift#L27